README.md

```diff
@@ -150,9 +150,15 @@ If you cannot find suitable test data on this repository, please contact us on t
       * alz.ccs.fl.NEB_5p--NEB_Clontech_3p.flnc.clustered.singletons.merged.aligned_tc.bed: first set of gene models generated by TAMA collapse
       * alz.ccs.fl.NEB_5p--NEB_Clontech_3p.flnc.clustered.singletons.merged.aligned_tc.2.bed: first set of gene models generated by TAMA collapse
     * fasta: 
+      * alz.ccs.fasta: CCS reads generated using pbccs on alz.bam in fasta format
+      * alz.ccs.fasta.gz: CCS reads generated using pbccs on alz.bam in gziped fasta format
       * primers.fasta: NEB Clonetech primers 
     * fastq:
+      * alz.ccs.fastq: CCS reads generated using pbccs on alz.bam in fastq format
+      * alz.ccs.fastq.gz: CCS reads generated using pbccs on alz.bam in gziped fastq format
       * test_hifi.fastq.gz: Reads mapping to a randomly selected contig from the whole genome assembly by [Cheng et al., 2021](https://www.nature.com/articles/s41592-020-01056-5) of the child of the GIAB Ashkenazim trio [RM8392](https://www-s.nist.gov/srmors/view_detail.cfm?srm=8392). The reads were taken from [SRR10382244](https://www.ncbi.nlm.nih.gov/sra/?term=SRR10382244).
+    * xml:
+      * alz.ccs.consensusreadset.xml: CCS reads generated using pbccs on alz.bam in consensusreadset.xml
```

Comment:
Then it should be removed from the readme, right?
```suggestion


```

Reply:
Indeed, I forgot to remove it from the Readme. 
Sorry for that.